### PR TITLE
feat(YALB-234): Add and install atomic theme.

### PR DESCRIPTION
**Pull request by nJim**
### Description of work
- Adds the yalesites-org/atomic project to the yalesties install profile
- Install the atomic theme
- Uninstall the bartik theme
- Rebuilds configuration for the theme blocks. A few random ones have also updated, but they have no functional changes.

### Functional testing steps:
- [ ] Visit the site to verify atomic is enabled and bartik is disabled. See the JIRA ticket for username and password. https://yalb234-yale-integration-env.pantheonsite.io/ 
- [ ] Review the starter theme code https://github.com/yalesites-org/atomic

### CODE REVIEW

**Review by ModulesUnraveled — APPROVED** [review]
I think this all looks good. I don't see anything obviously wrong.

Not sure what the build error is about though

File: config/sync/linkit.linkit_profile.default.yml [hunk 1]
```diff
@@ -6,18 +6,18 @@ dependencies:
     - node
```
Comment (vinmassaro) [inline config/sync/linkit.linkit_profile.default.yml:6]: Why does this config change as part of this PR?

File: config/sync/views.view.redirect.yml [hunk 2]
```diff
@@ -17,122 +17,12 @@ base_table: redirect
 base_field: rid
```
Comment (vinmassaro) [inline config/sync/views.view.redirect.yml:17]: Why does this config change as part of this PR?

**Review by vinmassaro — COMMENTED** [review]
Just question on why config changes for two files in the PR. Thanks!

File: config/sync/linkit.linkit_profile.default.yml (hunk 1 above)
Comment (nJim) [reply config/sync/linkit.linkit_profile.default.yml:6]: Gremlins. This is a recurring issue with config management where things magically change. Sometimes its as small as single quotes becoming double quotes. Other times it's new values appearing in the YAML files. 

The biggest reasons is that sometimes active configuration (the settings in the database) are incomplete. Then in the future another module installs, or cron runs, or something clicks the 'save' button on a related config form and now the active configuration as new values.

Expect this to happen semi regularly. Question it every time. In this case, the updated YALM files (staged config) correctly match what is in the database (active config) so the change is appropriate -- even though it is unrelated.

Comment (nJim) [reply config/sync/linkit.linkit_profile.default.yml:6]: Features was notorius for this kind of stuff; with zeros becoming false or values appearing or disappearing. Config management in D8/9 is much better but still has the occasional gremlin.

Comment (vinmassaro) [reply config/sync/linkit.linkit_profile.default.yml:6]: Thanks, that's what I assumed and wanted to double check.

**Pull request merged**